apps/backend/src/deposits/entities/pos-deposit.entity.ts

```diff
@@ -43,12 +52,23 @@ export class POSDepositEntity {
   @Column({ default: false })
   match: boolean;
 
+  // rename this to just payment id?
   @Column({ nullable: true })
   matched_payment_id?: string;
 
-  method?: number;
+  @ManyToOne(() => PaymentMethodEntity, (pd) => pd.method)
+  @JoinColumn({ name: 'card_vendor' })
+  payment_method: PaymentMethodEntity;
 
   constructor(data?: TDI34Details) {
     Object.assign(this, data?.resource);
   }
+
+  public get timestamp(): Date {
```

Comment:
use `getters` and `setters` to manipulate, cast, format data etc.. 

it is better to do it here than do casting in every logic where the property is used.